app/helpers/home_helper.rb

```diff
@@ -0,0 +1,16 @@
+module HomeHelper
+    def randomText
```

Comment:
Ruby-ban a snake_case-t szoktuk használni, a camelCase helyett.
```suggestion
    def random_text
```